lib/chef/mixin/which.rb

```diff
@@ -19,6 +19,10 @@ class Chef
   module Mixin
     module Which
       def which(cmd, opts = {})
+        Chef::Mixin::Which.which(cmd, opts)
+      end
+
+      def self.which(cmd, opts = {})
```

Comment:
i think you just triple-negative'd my brain into a loop. This definitely works; but I don't really know enough about `include`/`extend` to say definitively that it's worse.